lib/cisco_node_utils/bgp.rb

```diff
@@ -84,7 +84,9 @@ def self.enabled
     end
 
     def self.enable(state='')
-      config_set('bgp', 'feature', state: state)
+      # IOS XR has no '[no] feature xyz'.
+      config_set('bgp', 'feature', state: state) if
+        node.client.api == 'NXAPI' || state == 'no'
```

Comment:
This hits the 'feature' block of yaml amd in the XR no case only, I do a 'no router bgp'. So I am using it in a way. Otherwise changing the ASN testcases don't work well.